<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>779</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
       background-color: #f0f0f0; /* Thay đổi màu nền cho MainWindow */
   }</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>911</width>
      <height>761</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>640</y>
        <width>891</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="clear">
         <property name="styleSheet">
          <string notr="true">QPushButton {
                              background-color: #4CAF50; /* Màu xanh lá cây */
                              border: none;
                              color: white;
                              font: 16px &quot;Segoe UI&quot;;
                              border-radius: 5px;
                              padding: 10px 20px; /* Thêm padding */
                          }
                          QPushButton:pressed {
                              background-color: #45a049; /* Màu khi nhấn */
                          }</string>
         </property>
         <property name="text">
          <string>Clear Text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_2">
         <property name="styleSheet">
          <string notr="true">QPushButton {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(116, 116, 116), stop: 1 rgba(50, 50, 50));
                              border: 15px;
                              color: rgb(255, 255, 255);
                              font: 20pt &quot;Segoe MDL2 Assets&quot;;
                              border-radius: 10px;
                          }
                          QPushButton:pressed {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(50, 50, 50), stop: 1 rgba(116, 116, 116));
                          }</string>
         </property>
         <property name="text">
          <string>Clear Character</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startcam">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
                              background-color: #008CBA; /* Màu xanh dương */
                              border: none;
                              color: white;
                              font: 16px &quot;Segoe UI&quot;;
                              border-radius: 5px;
                              padding: 10px 20px; /* Thêm padding */
                          }
                          QPushButton:pressed {
                              background-color: #007B9E; /* Màu khi nhấn */
                          }</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pausecam">
         <property name="styleSheet">
          <string notr="true">QPushButton {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(116, 116, 116), stop: 1 rgba(50, 50, 50));
                              border: 15px;
                              color: rgb(255, 255, 255);
                              font: 20pt &quot;Segoe MDL2 Assets&quot;;
                              border-radius: 10px;
                          }
                          QPushButton:pressed {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(50, 50, 50), stop: 1 rgba(116, 116, 116));
                          }</string>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>891</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="camera1">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="camera2">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>482</y>
        <width>751</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="text1">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedArabEmirates"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="text2">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedArabEmirates"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>470</y>
        <width>131</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="space">
         <property name="styleSheet">
          <string notr="true">QPushButton {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(116, 116, 116), stop: 1 rgba(50, 50, 50));
                              border: 15px;
                              color: rgb(255, 255, 255);
                              font: 20pt &quot;Segoe MDL2 Assets&quot;;
                              border-radius: 10px;
                          }
                          QPushButton:pressed {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(50, 50, 50), stop: 1 rgba(116, 116, 116));
                          }</string>
         </property>
         <property name="text">
          <string>SPACE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="check">
         <property name="styleSheet">
          <string notr="true">QPushButton {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(116, 116, 116), stop: 1 rgba(50, 50, 50));
                              border: 15px;
                              color: rgb(255, 255, 255);
                              font: 20pt &quot;Segoe MDL2 Assets&quot;;
                              border-radius: 10px;
                          }
                          QPushButton:pressed {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(50, 50, 50), stop: 1 rgba(116, 116, 116));
                          }</string>
         </property>
         <property name="text">
          <string>CHECK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="send">
         <property name="styleSheet">
          <string notr="true">QPushButton {
                              background-color: #f44336; /* Màu đỏ */
                              border: none;
                              color: white;
                              font: 16px &quot;Segoe UI&quot;;
                              border-radius: 5px;
                              padding: 10px 20px; /* Thêm padding */
                          }
                          QPushButton:pressed {
                              background-color: #d32f2f; /* Màu khi nhấn */
                          }</string>
         </property>
         <property name="text">
          <string>SEND</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="img_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>441</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>891</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
     <widget class="QLabel" name="img_label_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>441</width>
        <height>441</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>600</y>
        <width>881</width>
        <height>24</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="record_button">
       <property name="text">
        <string>Start Record</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stop_record_button">
       <property name="text">
        <string>Stop Record</string>
       </property>
      </widget>
      <widget class="QPushButton" name="send_video">
       <property name="text">
        <string>Send Video</string>
       </property>
      </widget>
      <widget class="QPushButton" name="play_video">
       <property name="text">
        <string>Play Video</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
